frontend/src/components/Signup.jsx

```diff
@@ -16,7 +16,7 @@ import {
 import { ArrowLeft, CheckCircleFill, Eye, EyeSlash, ExclamationTriangleFill } from "react-bootstrap-icons";
 
 //Styles
-import "../css/Signup.css";
+import "../css/common.css";
```

Comment:
Centralizing common styles in a "common.css" is a good practice, ensuring consistent styling across components.